storybook/pages/MembersSelectorPage.qml

```diff
@@ -90,42 +90,15 @@ SplitView {
             }
         }
 
-        readonly property var temporaryModel: ListModel {
-            Component.onCompleted: usersStoreMock.resetTemporaryModel()
-        }
-
-        function appendTemporaryModel(pubKey, displayName) {
-            temporaryModel.append({
-                                      pubKey: pubKey,
-                                      displayName: displayName,
-                                  })
-        }
-
-        function removeFromTemporaryModel(pubKey) {
-            for(let i = 0; i < temporaryModel.count; i++) {
-                if (temporaryModel.get(i).pubKey === pubKey) {
-                    temporaryModel.remove(i, 1)
-                    return
-                }
-            }
-        }
-
-        function resetTemporaryModel() {
-            temporaryModel.clear()
-            for(let i = 0; i < usersModel.count; i++) {
-                const obj = usersModel.get(i)
-                temporaryModel.append(obj)
-            }
-        }
-
-        function updateGroupMembers() {
+        function updateGroupMembers(membersPKHash) {
+            const pubKeys = membersPKHash.split(",")
             const users = []
-            for(let i = 0; i < temporaryModel.count; i++) {
-                const obj = temporaryModel.get(i)
+            for(let i = 0; i < pubKeys.count; i++) {
```

Comment:
Not sure how does it work. If `pubKeys` is array of strings, does it `count` property and `get` method?

Reply:
Solved!